samza-kafka/src/test/scala/org/apache/samza/config/TestKafkaConfig.scala

```diff
@@ -220,7 +220,6 @@ class TestKafkaConfig {
     val kafkaConfig = new KafkaConfig(mapConfig)
     assertEquals("3", kafkaConfig.getChangelogStreamReplicationFactor("store-with-override"))
     assertEquals("2", kafkaConfig.getChangelogStreamReplicationFactor("store-without-override"))
-    assertEquals("2", kafkaConfig.getDefaultChangelogStreamReplicationFactor)
```

Comment:
Can you please add unit tests for the new flows in `KafkaConfig` for RF?

Reply:
added in TestKafkaConfig and updated others